### PR TITLE
Fully implement Attnam Catacombs

### PR DESCRIPTION
**Pull request by red-kangaroo**
* New prisoners in Cellar.
* You can no longer steal vodka.
* Zombie of Khaz-zadm as a boss at the end of Catacombs.

### CODE REVIEW

**Comment by andrewtweber** [comment]
I wish there were a "jaw drop" reaction :)

File: Script/char.dat [hunk 1]
```diff
@@ -5332,6 +5403,7 @@ necromancer
   CanRead = true;
   CanBeGenerated = true;
   ClassStates = INFRA_VISION;
+  ConsumeFlags = CT_MEAT|CT_LIQUID|CT_PROCESSED|CT_BONE; /* Necromancers eat vegetarians, not vegetables! */
```
Comment (AlexMooney) [inline Script/char.dat]: TODO: Add Exalted Necromancer (otherwise normal necromancer who just ate a bunch of ommel bone)

File: Script/char.dat [hunk 2]
```diff
@@ -453,6 +453,23 @@ farmer
...
+    UsesLongAdjectiveArticle = true;    
+    Adjective = "eunuch";
+    CanBeGenerated = false;
+    AttributeBonus = -10;
+    RightWielded = 0;
+    HostileReplies == "\"You don't have the balls to kill me!\"";
+    FriendlyReplies =
+    {
+      3,
+      "\"Let�s just say that Petrus� wife No. 3 was beautiful and I wasn�t careful.\"",
```
Comment (red-kangaroo) [inline Script/char.dat]: Weird...

**Comment by AlexMooney** [comment]
I don't want to review this carefully, because I gotta try it for myself tonight! :laughing:

**Comment by AlexMooney** [comment]
Can we keep the spider level and the catacombs on separate PRs?  They're different dungeons.

**Comment by red-kangaroo** [comment]
@AlexMooney Sure, after I figure out how to revert commits and move them to a different branch. :)

@fejoa Well, I actually have no idea who Tom Waits is.

**Comment by emlai** [comment]
Bump. Is this ready for merge?

**Comment by emlai** [comment]
Should we merge this before it gets any bigger? 😄

**Comment by red-kangaroo** [comment]
I think I need to fix UT stairs after my adding/reverting Spider Level, plus I need to test a bit what have I broken, but otherwise, it's nearly complete. :) Other ideas can have their own pull request. :D

File: Script/dungeons/Attnam.dat [hunk 3]
```diff
@@ -282,50 +284,57 @@ Dungeon ATTNAM;
           f = fountain { SecondaryMaterial = ICE; }
           d = decoration(DOUBLE_BED);
           B = GOLD boulder(1);
-          t = christmastree;
```
Comment (red-kangaroo) [inline Script/dungeons/Attnam.dat]: Oh, sorry, I totally missed this. :/

**Pull request merged**